include/seqan3/alphabet/cigar/cigar.hpp

```diff
@@ -72,14 +72,23 @@ class cigar : public alphabet_tuple_base<cigar, uint32_t, cigar_op>
     // Inherit constructors from base
     using base_t::base_t;
 
-    //!\copydoc alphabet_tuple_base::alphabet_tuple_base(component_type const alph)
+    /*!\brief Construction via a value of one of the components.
+     * \tparam component_type Must be one uniquely contained in the type
+                              list of the composite.
+     * \param  alph           The value of a component that should be assigned.
```

Comment:
why can't you use the \copydoc anymore? You can use copydoc AND give more details :)

Reply:
Hmmm I guess I could! That would also copy the qualified snippet over, though. Then there would be the qualified snippet example *and* the cigar example, and I think the reason for the issue was that the qualified one shouldn't be in the cigar documentation?